packages/tag/src/components/styledComponent/StyledTag.js

```diff
@@ -9,18 +9,25 @@ const typeStyles = {
   [tagTypes.SECONDARY]: css`
     background-color: ${colorConstants.INDIGO_MILK_CAP};
     color: white;
+    .icon {
+      color: white;
```

Comment:
We don't need to set `color` again if the color is the same used on the parent node... please remove it here and on line 29